include/singa/core/common.h

```diff
@@ -100,13 +100,21 @@ typedef struct _Context {
   std::mt19937 random_generator;
 #ifdef USE_CUDA
   cublasHandle_t cublas_handle;
-  cudaStream_t stream;
-  curandGenerator_t curand_generator;
+  cudaStream_t stream; 
+  curandGenerator_t curand_generator; 
+
 #ifdef USE_CUDNN
   cudnnHandle_t cudnn_handle;
 #endif
 #endif  // USE_CUDA
 
+#ifdef USE_DIST
```

Comment:
In the CPU only version, there is no cuda library. It will not have those cuda related headers.
In the building process, the default USE_DIST is 0.
So the Travis CI build of CPU version didn't include this part, you may check the CI CPU version build result
My concern not to use USE_CUDA instead is that it could add three more useless cuda stream into the single GPU version.

Reply:
All of them in Travis CI build are only CPU version
https://travis-ci.org/github/apache/singa/builds/693742421